spec/factories/cards.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :card do
+    original_text 'hello'
+    translated_text 'привет'
+    review_date  (DateTime.current.to_date)
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.